Gododir/gen.go

```diff
@@ -405,7 +406,9 @@ func managementVersion(c *do.Context) {
 func version(packageName string) {
 	versionFile := filepath.Join(packageName, "version.go")
 	os.Remove(versionFile)
-	template := `package %s
+	template := `// +build go1.7	
```

Comment:
Why the build constraint (not pushing back just wondering)?

Reply:
This build constraint was added to `version.go` some time ago by hand. But as `gen.go` gens that file and overwrites, well, I included this so there would be no diff.

Reply:
Makes sense.